.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,23 @@
 ### What Changes does this PR introduce?
```

Comment:
The answers to these questions must be entered on the PR page in github, this template shouldn't be edited.

Reply:
OK!